relinker/src/main/java/com/getkeepsafe/relinker/SystemLibraryLoader.java

```diff
@@ -24,6 +25,7 @@ public void loadLibrary(final String libraryName) {
         System.loadLibrary(libraryName);
     }
 
+    @SuppressLint ("UnsafeDynamicallyLoadedCode")
```

Comment:
this is to address the lint warning below of code that is a calculated usage of `load`